great_expectations/rule_based_profiler/util.py

```diff
@@ -79,7 +78,7 @@ def get_batch_ids(
 
 
 def build_batch_request(
-    batch_request: Optional[Union[dict, str]] = None,
+    batch_request: Optional[Union[BatchRequest, RuntimeBatchRequest, dict]] = None,
```

Comment:
@cdkini Where there used to be str as a type for batch_request it must remain. This is used for obtaining it from variables. Could you please restore it -- in addition to the other types, (BatchRequest, RuntimeBatchRequest, dict), wherever it used to be? Thanks!